### DISCUSSION TITLE
Setting the border to a specific side does not appear

### DISCUSSION THREAD

**Discussion by RealCrayfish** [tailwindlabs/tailwindcss#8067] [Help]
I'm trying to create a "breaker" using a single side border on the bottom of my container. When I set the values nothing appears.

Heres my code:
```html
<div class="ml-52 mt-2">
    <div class="border-red-500 border-b border-solid">
        <h1 class="text-3xl">App</h1>
    </div>
</div>
```
And heres the output on the page:
![image](https://user-images.githubusercontent.com/77017028/162527450-23e1106f-c227-4c23-9a3e-0be4e5f8d558.png)

**Comment by cannap** [comment]
Hi
here it works 
https://play.tailwindcss.com/BNZ4E4mFJb 

maybe you override the colors not sure what does the devtools say?
